products/dns/api.yaml

```diff
@@ -81,7 +81,8 @@ objects:
             name: 'defaultKeySpecs'
             description: |
               Specifies parameters that will be used for generating initial DnsKeys
-              for this ManagedZone. Output only while state is not OFF
```

Comment:
Is it still important to note that it's "Output only while state is not OFF"?

Reply:
Added a comment to add a note back into the descriptions, since it isn't relevant until we have update support.